clients/cmd/docker-driver/config.go

```diff
@@ -153,7 +151,7 @@ func parseConfig(logCtx logger.Info) (*config, error) {
 	clientConfig.URL = flagext.URLValue{URL: url}
 
 	// parse timeout
-	if err := parseDuration(cfgTimeoutKey, logCtx, func(d time.Duration) { clientConfig.Timeout = d }); err != nil {
+	if err := parseDurationWithDefault(cfgTimeoutKey, logCtx, func(d time.Duration) { clientConfig.Timeout = d }, 10*time.Second); err != nil {
```

Comment:
why not just set the default on the config struct ?

Reply:
You mean [here](https://github.com/grafana/loki/blob/a2b66d35bdc29c54e44e1d5f5eb56536be98a41a/clients/pkg/promtail/client/config.go#L60)? That's also used by Promtail.